### DISCUSSION TITLE
Add examples of `DressingSelect` and `DressingInput` filetypes in autocmd

### DISCUSSION THREAD

**Discussion by hinell** [stevearc/dressing.nvim#85] [Ideas]
```
DressingSelect
DressingInput
```

I request to document the above file types and specify how can they  be used in the `autocmd`.  I'm currently using this filetype to ignore dynamic `relativenumber` switching ([see example here](https://www.reddit.com/r/neovim/comments/10bmy9w/lets_see_your_status_columns/)).

**Comment by stevearc** [comment]
That linked reddit post has nearly 90 comments. I have no idea which of those you're trying to copy.

The filetypes are just normal filetypes, you use them like you normally would with an autocmd. What kind of documentation are you thinking?

**Comment by hinell** [comment]
@stevearc I'm just proposing to mention in docs the fact they exist.
